boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -259,6 +259,13 @@ found in :ref:`nordic_segger_flashing`. Then build and flash
 applications as usual (see :ref:`build_an_application` and
 :ref:`application_run` for more details).
 
+.. note::
+
+   Flashing and debugging applications on the nRF5340 DK requires
+   upgrading the nRF Command-Line Tools to version 10.12.0. Further
+   information on how to install the nRF Command-Line Tools may be
+   found in :ref:`nordic_segger_flashing`.
```

Comment:
```suggestion
   Flashing and debugging applications on the nRF5340 DK requires
   upgrading the nRF Command Line Tools to version 10.12.0. Further
   information on how to install the nRF Command Line Tools can be
   found in :ref:`nordic_segger_flashing`.
```

Reply:
@ru-fu thanks, I will apply that.
Just, FYI, I chose to use dash to comply with how it is spelled in the corresponding doc page: https://docs.zephyrproject.org/latest/guides/tools/nordic_segger.html#nordic-segger

Might want to change that too?

Reply:
done now.